src/trio/_tests/test_socket.py

```diff
@@ -186,7 +186,10 @@ def interesting_fields(
         ) -> tuple[
             AddressFamily,
             SocketKind,
-            tuple[str, int] | tuple[str, int, int] | tuple[str, int, int, int],
+            tuple[str, int]
+            | tuple[str, int, int]
```

Comment:
Actually I don't think this `tuple[str, int, int]` is necessary?

Reply:
Indeed, thanks!